e2e/keywords/backing_image.resource

```diff
@@ -0,0 +1,24 @@
+*** Settings ***
+Documentation    Backing Image Keywords
+
+Library    ../libs/keywords/backing_image_keywords.py
+
+*** Keywords ***
+Create backing image ${backing_image_name} with
+    [Arguments]    &{config}
+    create_backing_image    ${backing_image_name}    &{config}
+
+Verify all disk file status of backing image ${backing_image_name} are ready
+    all_disk_file_status_are_ready    ${backing_image_name}
+
+Verify try to clean up backing image ${backing_image_name} from a disk will fail
+    Run Keyword And Expect Error    *    clean_up_backing_image_from_a_random_disk    ${backing_image_name}
+
+Verify try to delete backing image ${backing_image_name} will fail
+    Run Keyword And Expect Error    *    delete_backing_image    ${backing_image_name}
+
+Clean up backing image ${backing_image_name} from a disk
```

Comment:
Yes, it's possible. We can retrieve the node to know the uuid of the disk, and then clean up the backing image by uuid.

But maybe it's not required at this moment, since the test case is testing the cleaning up API works or not, specifying a node add no extra information to it.